packages/cozy-client/src/withMutations.jsx

```diff
@@ -24,9 +33,16 @@ const withMutations = (mutations = {}) => WrappedComponent => {
           saveDocument: client.save.bind(client),
           deleteDocument: client.destroy.bind(client)
         },
-        ...(typeof mutations === 'function'
-          ? mutations(client, props)
-          : mutations)
+        ...reduce(
+          mutations.map(
```

Comment:
Since passing an object to `withMutations` would no longer work, isn't this a breaking change? Or am I missing something?

Reply:
I thought the same but in fact, with the ... syntax, it will work since mutations here is all the mutations and they are merge, you are not passing an array to withMutations, only allowing to pass multiple mutations to be passed as parameters.